wayfire.ini

```diff
@@ -73,11 +76,14 @@ plugins = \
 # https://github.com/WayfireWM/wayfire/wiki/Configuration#blur
 
 # Close focused window.
-close_top_view = <super> KEY_W | <super> BTN_MIDDLE | <alt> KEY_F4
+close_top_view = <super> KEY_W | <super> BTN_MIDDLE | <alt> KEY_FN_F4 | <alt> KEY_F4
```

Comment:
Does this depend on the keyboard model or something?

Reply:
some keyboards directly use fn keys without needing to use fn + f1 etc